doc/rest-api.md

```diff
@@ -754,6 +755,35 @@ Input (none at present):
 
 HTTP code for this should be 202 (Accepted).
 
+## `/1.0/containers/<name>/console`
+### POST
+ * Description: attach to a container's console devices
+ * Authentication: trusted
+ * Operation: async
+ * Return: standard error
+
+Input (attach to /dev/console):
+
+    {
+        "width": 80,                    # Initial width of the terminal (optional)
+        "height": 25,                   # Initial height of the terminal (optional)
+        "tty": 0,                       # Console device to attach to
+        "escape": a                     # Escape character to use when detaching from the console via "Ctrl+<escape> q"
```

Comment:
Not sure this makes sense at the API level. With LXD you'd think we would have the detach sequence disabled, have the CLI tool implement the "ctrl+<escape> q" handling and then just have it close the websocket sessions when that happens.